backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/RemoveImageCommandCallback.java

```diff
@@ -0,0 +1,82 @@
+package org.ovirt.engine.core.bll.storage.disk.image;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.enterprise.inject.Typed;
+import javax.inject.Inject;
+
+import org.ovirt.engine.core.bll.tasks.CommandCoordinatorUtil;
+import org.ovirt.engine.core.bll.tasks.interfaces.CommandCallback;
+import org.ovirt.engine.core.common.action.RemoveImageParameters;
+import org.ovirt.engine.core.common.businessentities.AsyncTaskStatus;
+import org.ovirt.engine.core.compat.CommandStatus;
+import org.ovirt.engine.core.compat.Guid;
+import org.ovirt.engine.core.vdsbroker.vdsbroker.VDSErrorException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Typed(RemoveImageCommandCallback.class)
+public class RemoveImageCommandCallback implements CommandCallback {
+    private static final Logger log = LoggerFactory.getLogger(RemoveImageCommandCallback.class);
+
+    @Inject
+    private CommandCoordinatorUtil commandCoordinatorUtil;
+
+    @Override
+    public void doPolling(Guid cmdId, List<Guid> childCmdIds) {
+        RemoveImageCommand<RemoveImageParameters> command = getRemoveImageCommand(cmdId);
+        Set<Guid> taskIds = new HashSet<>(command.getReturnValue().getVdsmTaskIdList());
+        Map<Guid, AsyncTaskStatus> idToTaskStatusMap = commandCoordinatorUtil.pollTasks(taskIds);
+        for (Map.Entry<Guid, AsyncTaskStatus> idToTaskStatus : idToTaskStatusMap.entrySet()) {
+            Guid id = idToTaskStatus.getKey();
+            AsyncTaskStatus status = idToTaskStatus.getValue();
+            if (status.getTaskIsRunning()) {
+                log.info("Waiting on remove image command to complete the task (taskId = '{}')", id);
```

Comment:
`to complete the task (taskId = '{}')` --> `to complete the task '{}'`

Reply:
@barpavel this is a classic case to propose a change using a 'suggestion' like:
```suggestion
                log.info("Waiting on remove image command to complete the task '{}'", id);
```

Reply:
Wow, I didn't know about this nice feature, thanks!
After some searching I found `Add a suggestion` icon. Next time I'll use it :)
But if you change it directly in the PR in GitHub, need to take care not to override it back if committing more changes.... Right?

Reply:
right, it adds another commit so you need to rebase and get it on your local branch before committing more changes